src/Component/InputBox.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from 'styled-components';
+import { useMessageDispatch } from '../Context/MessageContext';
+import useInput from '../Hooks/useInput';
+
+// useMessageState
+
+const InputBox = ( ) => {
+    //const state = useMessageState();
+    const dispatch = useMessageDispatch();
+    const { input , onInputChange, resetInput } = useInput('');
+
+    const onSubmit = ( e : React.FormEvent ): void => {
+        e.preventDefault();
+        if (!input){
+            alert('메세지를 입력하세요 !');
+        }
+        else if(dispatch) {
+            dispatch({
+                type: 'ADD'
+            });
+        }
+        resetInput();
+    }
+
+    return <>
+        <InputArea>
+        <InputField
+            value = { input }
+            onChange = { onInputChange }
+            placeholder = "Text your message here."
+        />
+        <InsertBtn onClick={onSubmit}>
+        +
+        </InsertBtn>
+        </InputArea></>
```

Comment:
이 부분도 fragment 사용하지 않으셔도 됩니다.

```suggestion
    return (
    <InputArea>
      <InputField
        value={input}
        onChange={onInputChange}
        placeholder="Text your message here."
      />
      <InsertBtn onClick={onSubmit}>+</InsertBtn>
    </InputArea>
  );
```